packages/core/test/predictableExec.test.ts

```diff
@@ -424,11 +415,14 @@ describe('predictableExec', () => {
           entry: sendTo('myChild', { type: 'KNOCK_KNOCK' }),
           invoke: {
             id: 'myChild',
-            src: fromCallback(({ receive }) => {
```

Comment:
Currently, a callback actor couldn't be used here for this test case because the event sent from an entry action to an invoked actor is being received by it *before* a callback actor has a chance to setup listeners. 

It's a timing issue that we should fix but that's outside of the scope of this PR.